### ISSUE TITLE
add k8sgpt-operator

### ISSUE THREAD

**Issue by jkleinlercher** [suxess-it/sx-cnp-oss#18]

**Comment by phac008** [comment]
do we still need that? if so, should we put that on a roadmap?

**Comment by jkleinlercher** [comment]
Good point, maybe we should put this task in our „evaluate AI supported operations“ story to support the „AI integration“ roadmap topic

**Comment by jkleinlercher** [comment]
maybe part of an AI integration capability
